src/pyhf/probability.py

```diff
@@ -42,17 +38,22 @@ def log_prob(self, value):
 
 
 class Simultaneous(object):
-    def __init__(self, pdfobjs, indices):
-        self.tv = TensorViewer(indices)
+    def __init__(self, pdfobjs, tv, batch_size):
```

Comment:
Would prefer a more explicit argument name for the TensorViewer here than `tv`, but not the end of the world. Leaving `self.tv` is fine though, just for the class init would be nice.